aten/src/ATen/cpu/vec256/vec256_double.h

```diff
@@ -106,9 +106,15 @@ template <> class Vec256<double> {
   Vec256<double> sin() const {
     return map(std::sin);
   }
+  Vec256<double> sinh() const {
+    return map(std::sinh);
+  }
   Vec256<double> cos() const {
     return map(std::cos);
   }
+  Vec256<double> cosh() const {
+    return map(std::cos);
```

Comment:
Those functions are in sleef, can we use them instead of `std` ones?

Reply:
No, because they don't cover the full range. We'll need to create something like cosh_u10 and then write a kernel or add a flag that deals with that.